docs/CloudPermissions.md

```diff
@@ -0,0 +1,56 @@
+# Restricting cloud privileges for deploying Scalar DL
+
+This guide explains how to restrict privileges for deploying Scalar DL on Azure.
+
+In general from security perspective, it is better to assign restricted privileges to users not to make them able to do operations that they are not supposed to do.
+
+## Azure
+
+The following JSON is a custom role that allow users to manage resources for Scalar DL. This is sufficient but not necessary.
+
+In Azure Portal, you can create the role in Subscriptions section. Choose your subscription and select Access control (IAM) from the menu, then click `+Add` and select `Add custom role`. 
+Once the role is created, you can assign it to a user or a group from `Add role assignment` in the `+Add` menu.
+
+Please keep your subscription ID in the `assignableScopes` array. You can give `rolename` and `description` as per your convenience.
```

Comment:
rolename or roleName?

Reply:
Sorry, its roleName changes accordingly in c601da6